models/object.py

```diff
@@ -384,11 +393,11 @@ def copy_object(editor):
 
             # Store the object that was clicked on in the editor's clipboard
             ob = editor.object[o]
-            name = ob.objectName()+'(1)'
-            if ob.childWidget.type == 'image':
-                editor.clipboard_object = ClipboardObject(name,ob.frameGeometry().width(), ob.frameGeometry().height(), ob.path,  ob.type)
+            undo_name = ob.objectName()+'(1)'
+            if ob.object_type == 'image':
+                editor.clipboard_object = ClipboardObject(ob.childWidget.frameGeometry().width(), ob.childWidget.frameGeometry().height(), ob.childWidget.path, ob.object_type, undo_name) # TODO: move name
```

Comment:
not very important but imo it looks cleaner to have the name at the end of the constructor, so that the "ob" method calls come first